website/blog/2025-11-4-boost-your-performance-with-search-bar.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Leverage your productivity using Search Bar
+description: Boost your performance with Search Bar component
+authors: [gastoner]
+tags: [podman-desktop, features]
+hide_table_of_contents: false
+---
+
+import ReactPlayer from 'react-player'
+import ThemedImage from '@theme/ThemedImage';
+
+In 1.22 Podman Desktop release we have introduced new Search Bar component. This was further enhanced in 1.23 release with handy features.
+
+## Search Bar
+
+The Search Bar (previously known as Command Palette) is a powerful, keyboard-first feature that brings everything in Podman Desktop just a few keystrokes away. Located directly in the title bar, it provides unified access to commands, documentation, and navigation across your entire workspace.
+
+<ReactPlayer src='https://www.youtube.com/watch?v=tvcXlrmO08Y' width="100%" height="500px"  />
```

Comment:
Why have we included this community meeting recording? It does not cover anything related to the blog.